SpywareFilter/sections/whitelist.txt

```diff
@@ -306,7 +306,7 @@ extrahop.com#%#//scriptlet('prevent-setTimeout', 'ad blocker')
 ! www.satofull.jp images not shown
 @@||rtoaster.jp/Rtoaster.js$domain=satofull.jp
 ! https://github.com/AdguardTeam/AdguardFilters/issues/111052
-@@||wcs.naver.net/wcslog.js$domain=ciokorea.com|adpiamall.com|live.joins.com|naver.com|jbe.go.kr|privatenote.co.kr
+@@||wcs.naver.net/wcslog.js$domain=ciokorea.com|adpiamall.com|live.joins.com|naver.com|jbe.go.kr|privatenote.co.kr|dabangapp.com|kyobobook.co.kr
```

Comment:
Ok, but why do you need an exception for `|kyobobook.co.kr`?

Reply:
I checked on Mozilla Firefox Dev 102.0b9 with AdGuard Browser Extension 4.0.181.